tools/generate_config.py

```diff
@@ -0,0 +1,326 @@
+#!/usr/bin/env python3
+
+import argparse
+from pathlib import Path
+import json
+try:
+    import jsonschema
+except ImportError:
+    jsonschema = None
+from collections import OrderedDict
+
+THIS_DIR = Path(__file__).parent
+
+if (THIS_DIR / 'schemas').exists():
+    SCHEMAS_DIR = THIS_DIR / 'schemas'
+else:
+    SCHEMAS_DIR = THIS_DIR.parent / 'share' / 'schemas'
+
+ENCLAVE_CFG_SCHEMA_PATH = SCHEMAS_DIR / 'enclave-config.schema.json'
+
+# Validates a particular enclave_cfg_path against ENCLAVE_CFG_SCHEMA_PATH
+def validate(args):
```

Comment:
This is already implemented as `sgx-lkl-cfg validate`.

Reply:
You marked this as resolved, what's the resolution?

Reply:
I removed it.